common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -231,6 +231,20 @@ def test_cant_submit_blank_answer(self):
         )
         self.assertIn('is-disabled', self.problem_page.q(css='div.problem button.check').attrs('class')[0])
 
+    @attr('shard_7')
```

Comment:
@alisan617 this syntax to specify a shard is not correct-- look at the other test points.

This test point is running and passing, but in shard 9, which is where all test points that don't specify a shard runs.

https://build.testeng.edx.org/job/edx-platform-test-subset/555400/console

@staubina It's very interesting that this test is passing on all problem types. Perhaps there is something funky with your branch?